libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2285,7 +2285,8 @@ bool WaveTrack::Get(size_t iChannel, size_t nBuffers,
    std::optional<TrackIter<const WaveTrack>> iter;
    auto pTrack = this;
    if (pOwner) {
-      iter.emplace(TrackList::Channels(this).first.advance(iChannel));
+      const auto ppLeader = TrackList::Channels(this).first;
+      iter.emplace(ppLeader.advance(IsLeader() ? 0 : 1));
```

Comment:
What if `Get` is called on a leader track with `iChannel == 1` and `nBuffers == 1` ? It seems to me that this would still get the left-channel samples. My best guess is that `iter.emplace(ppLeader.advance(IsLeader() ? iChannel : 1)`.